ddtrace/utils/hook.py

```diff
@@ -0,0 +1,230 @@
+import sys
```

Comment:
If this is copy/pasta from `wrapt`, then we should have a comment here on that. GitHub link to tag/commit hash version of the file we copied, along with a description of changes we have made (maybe inline comments about what we have changed?)